pkg/splunk/enterprise/util.go

```diff
@@ -902,7 +902,7 @@ func updateAuxPhaseInfo(appDeployInfo *enterpriseApi.AppDeploymentInfo, desiredR
 }
 
 // changePhaseInfo changes PhaseInfo and AuxPhaseInfo for each app to desired state
-func changePhaseInfo(desiredReplicas int32, appSrc string, appSrcDeployStatus map[string]enterpriseApi.AppSrcDeployInfo) {
+func changePhaseInfo(currentReplicas, desiredReplicas int32, appSrc string, appSrcDeployStatus map[string]enterpriseApi.AppSrcDeployInfo) {
```

Comment:
Nit: You may want to name it as `adjustAfwPhaseInfoForScaleup()` or something like that.